flowauth/frontend/src/UserAdminDetails.js

```diff
@@ -117,7 +117,7 @@ class UserAdminDetails extends React.Component {
     return (
       <React.Fragment>
         <Grid xs={12}>
-          <Typography variant="headline" component="h1">
```

Comment:
This is because headline etc. are deprecated, right? Do we need to [explicitly turn on the hx variants](https://material-ui.com/style/typography/#migration-to-typography-v2)?